### DISCUSSION TITLE
Model pipeline buttons

### DISCUSSION THREAD

**Discussion by zethfoxster** [lllyasviel/stable-diffusion-webui-forge#2498] [Q&A]
is there a way to save defaults for the vae/diffusion settings/gpu ect... so if i click the "flux" button it automatically populates these with the saved settings so i dont have to remember which encoder/vae/diffusion ect ect for the 6 or 7 model types?
does this exist?

**Comment by abzaloff** [comment]
> is there a way to save defaults for the vae/diffusion settings/gpu ect... so if i click the "flux" button it automatically populates these with the saved settings so i dont have to remember which encoder/vae/diffusion ect ect for the 6 or 7 model types? does this exist?

There is in the model settings. Specify the required model loading parameters in the model settings.

**Reply by zethfoxster** [reply]
where exactly is this option, sorry ive looked in settings and not seen it
